exads/examples/prebidJS-client-RTB-InStreamVideo.html

```diff
@@ -0,0 +1,220 @@
+            user_id = localStorage.getItem('prebidJS.user_id');
+        }
+
+        let ip_script = document.createElement('script');
+        ip_script.type = 'text/javascript';
+        ip_script.src = 'https://api.ipify.org?format=jsonp&callback=userIpCallback';
+        document.getElementsByTagName('head')[0].appendChild(ip_script);
+
+        var pbjs = pbjs || {};
+        pbjs.que = pbjs.que || [];
+
+        handleRTB();
+
+        if (isEnabledDebug) {
+            onScriptLoaded();
+        }
+
+        function onScriptLoaded() {
+            pbjs.setConfig({
+                debug: isEnabledDebug,
+                cache: { url: 'https://prebid.adnxs.com/pbc/v1/cache' },
+                consentManagement: {
+                    gdpr: {
+                        cmpApi: 'static',
+                        timeout: 1000000,
+                        defaultGdprScope: true,
+                        consentData: {
+                            getTCData: {
+                                tcString: consentString,
+                                gdprApplies: isGDPRRequested
+                            }
+                        }
+                    }
+                }
+            });
+        }
+
+        // ======== DO NOT EDIT BELOW THIS LINE =========== //
+        function userIpCallback(user_ip) {
+            userIpLoaded = true;
+            if (user_ip.hasOwnProperty('ip')) userIp = user_ip.ip;
+        }
+
+        // MurmurHash3 hash function
+        function hashCode(str, seed = 0) {
+            let h1 = 0xdeadbeef ^ seed, h2 = 0x41c6ce57 ^ seed;
+            for (let i = 0, ch; i < str.length; i++) {
+                ch = str.charCodeAt(i);
+                h1 = Math.imul(h1 ^ ch, 2654435761);
+                h2 = Math.imul(h2 ^ ch, 1597334677);
+            }
+            h1 = Math.imul(h1 ^ (h1 >>> 16), 2246822507) ^ Math.imul(h2 ^ (h2 >>> 13), 3266489909);
+            h2 = Math.imul(h2 ^ (h2 >>> 16), 2246822507) ^ Math.imul(h1 ^ (h1 >>> 13), 3266489909);
+            return 4294967296 * (2097151 & h2) + (h1 >>> 0);
+        }
+
+        function handleRTB() {
+            adUnits = [{
+                code: 'postbid_iframe',
+                mediaTypes: {
+                    video: {
+                        mimes: ['video/mp4'],
+                        protocols: [3, 6]
+                    }
+                },
+                bids: [{
+                    bidder: 'exadsadserver',
+                    params: {
+                        zoneId: zoneId,
+                        fid: fid,
+                        partner: partner,
+                        siteId: siteId,
+                        catIab: ['IAB25-3'],
+                        userIp: 0, // DO NOT EDIT - Client side data
+                        userId: user_id,
+                        impressionId: impression_id.toString(), // DO NOT EDIT - Custom hash
+                        stream: {
```

Comment:
stream is actually instream right? If yes, then we shouldn't call them stream, even though it can be used between instream, outstream and video slider

Reply:
Well, that info is for `imp` object, so your `stream` would become `imp` and then you would have `imp.ext.video_cta` 
Btw, `protocols` should be under `video` , that's wrong in our documentation, I already pinged guys to fix that in our documentation. 
It should be (inside `imp`):
```
video: {
  "mimes":['video/mp4'],
  "protocols":[3,6]
  },
"ext": {
"video_cta: 0
}
```

Reply:
fixed